gradle-service/src/test/java/com/epam/reader/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.reader.service;
+
+import org.hamcrest.core.IsEqual;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * Created by admin
+ * on 20.03.2016.
+ */
+public class ReaderServiceTest {
+    @Test
+    public void readMsgShouldGetCorrectMessageFromFile() throws IOException {
```

Comment:
You repeat yourself)
In my point of view you do not need the test for DAO layer)
Your unit is DAO + Service